submitit/helpers.py

```diff
@@ -331,14 +332,26 @@ def __init__(self) -> None:
         >>> torch.distributed.init_process_group(backend="nccl")
         >>> print(f"master: {dist_env.master_addr}:{dist_env.master_port}")
         """
-        self._job_env = JobEnvironment()
+        try:
+            self._job_env = JobEnvironment()
+        except RuntimeError as e:
+            if SlurmJobEnvironment._env["job_id"] in os.environ:
+                # identified a slurm env without submitit, so let's use it
```

Comment:
As discussed, this is a really weird use case and a surprising thing to try to fix: this is basically to make it possible for users (that are not using submitit to launch jobs) to use a helper function from submitit...